theme.css

```diff
@@ -4008,3 +4008,51 @@
   font-weight: 600;
 }
 
+
+/*
+ * Fix: Detailed page image missing for actor and season overview pages.
+ *
+ * Targets pages to unset the display: none that may have been applied
+ * over-zealously.
+ *
+ * Notes:
+ *  - "#itemDetailPage .detailPagePrimaryContainer:has(.btnPlay.hide)"
+ *    is the actor page, it's the only (needs checking) detail page
+ *    that has a hidden play button.
+ *  - "#itemDetailPage .detailPagePrimaryContainer:has(.nameContainer > h3.itemName:not(.focuscontainer-x)"
+ *    is the season overview page, it's the only (needs checking) detail
+ *    page where the header has a h3.itemName:not(.focuscontainer-x) element
+**/
+#itemDetailPage .detailPagePrimaryContainer:has(.btnPlay.hide) .card,
+#itemDetailPage .detailPagePrimaryContainer:has(.btnPlay.hide) .cardBox,
+#itemDetailPage .detailPagePrimaryContainer:has(.nameContainer > h3.itemName:not(.focuscontainer-x)) .card,
+#itemDetailPage .detailPagePrimaryContainer:has(.nameContainer > h3.itemName:not(.focuscontainer-x)) .cardBox {
+  display: block;
+}
+
+#itemDetailPage .detailPagePrimaryContainer:has(.btnPlay.hide),
+#itemDetailPage .detailPagePrimaryContainer:has(.nameContainer > h3.itemName:not(.focuscontainer-x)) {
+  width: 90%;
+}
```

Comment:
I think the icon-based selectors I mentioned in #15 are probably more stable in the long run then having to rely on play buttons and other nested selectors.

I think the less things we base the selectors on the better, since we can't control the code we're basing them on. A `span.class` is much less likely to change in the future (if ever) then nested selectors where the DOM could change in the next version and break the theme again.